src/components/Footer.jsx

```diff
@@ -1,5 +1,42 @@
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div
+      className="Footer"
```

Comment:
className is how you add CSS styles, it's not an HTML class
We're using TailwindCSS, look at the docs linked in the resources tab on discord to learn more about it
put everything into one div, use <p> tags for the text, and move all the stuff from styles into classname